seeder.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const quizName = process.argv[2];
+const quizesDir = fs.readdirSync('./seeds');
+const inquirer = require('inquirer');
+const chalk = require('chalk');
+const quizFolder = quizesDir.filter(currentFolder => {
+  return currentFolder === quizName;
+});
+inquirer
+  .prompt([
+    {
+      type: 'confirm',
+      name: 'inDocker',
+      message: 'Are you in pushkin-db docker?'
+    }
+  ])
+  .then(answer => {
+    if (answer.inDocker) {
+      inquirer
+        .prompt([
+          {
+            type: 'confirm',
+            name: 'ranMigration',
+            message: 'Have you ran your migrations?'
+          }
+        ])
+        .then(answer => {
+          if (answer.ranMigration) {
+            inquirer
+              .prompt([
+                {
+                  type: 'confirm',
+                  name: 'properSeeds',
+                  message: 'Have you properly fill out your seed CSV files?'
+                }
+              ])
+              .then(answer => {
+                if (answer.properSeeds) {
+                  const runSeed = require(`./seeds/${quizFolder}/index`);
+                  runSeed();
+                } else {
+                  console.log(
+                    chalk.red(
+                      'please make sure you have proper seeded CSV files before running this command'
```

Comment:
Please insure that you have correct CSV files before running this command

For more info go to https://github.com/l3atbc/pushkin-db